### DISCUSSION TITLE
Move completed habits to bottom

### DISCUSSION THREAD

**Discussion by naught101** [iSoron/uhabits#827] [Feature Requests] [answered]
I would like to have a clear visual distinction between complete and incomplete habits. I think the best way to do this would be to move completed habits to the bottom of the list, below a visual separator line. Maybe also grey them out, as mentioned at https://github.com/iSoron/uhabits/discussions/808

This is a better solution than hiding completed habits, because:

- it allows positive visual feedback of completion (habit moves to bottom, changes colour), 
- it allows immediate analysis of how many active habits are being completed per day, and 
- it allows completed habits to be over-completed (e.g. 6 days on a 5 per week habit). 
- It also allows a long habit list to still be practically useful, while retaining the ability to remind yourself what habits will be upcoming tomorrow.

**Comment by naught101** [comment]
It also makes it easier to undo a mistakenly completed habit, compared to when hiding is turned on.

**Comment by naught101** [comment, marked as answer]
Ah! It looks like this is already completed in 2.0! Cool! https://github.com/iSoron/uhabits/pull/660

I might give the alpha a go :)

**Reply by naught101** [reply]
Mm. Won't install on my phone. Not sure if that's because it's already installed via fdroid. Maybe I should just wait. Is there a release date planned yet?

**Reply by iSoron** [reply]
- F-Droid builds use a different key to sign the application, so it's not possible to upgrade directly. You will need to export a backup of your data, uninstall the current F-Droid version, install the alpha, then import the backup.
- We unfortunately don't have a planned release date.
